fiware-region-sanity-tests/tests/fiware_region_with_networks_tests.py

```diff
@@ -451,3 +454,77 @@ def test_deploy_instance_with_networks_and_e2e_snat_connection(self):
         self.assertEqual(expected_instance_name, received_hostname,
                          "Received hostname '%s' in PhoneHome request does not match with the expected instance name" %
                          received_hostname)
+
+    def test_deploy_instance_with_networks_and_check_metadata_service(self):
+        """
+        Test whether it is possible to deploy an instance with new network and check that metadata service
+         is working properly (PhoneHome service)
+        """
```

Comment:
I think the unnittest/nosetests lib only gets the first line of this description. The tooltip will use this data to show the description in the result report.